src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/ArrayExtension.cs

```diff
@@ -2,42 +2,35 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/***************************************************************************\
-*
-*
-*  Class for Xaml markup extension for Arrays
-*
-*
-\***************************************************************************/
-
 using System.Collections;
 using System.ComponentModel;
 using System.Runtime.CompilerServices;
 
 namespace System.Windows.Markup
 {
     /// <summary>
-    ///  Class for Xaml markup extension for Arrays.
+    /// Class for Xaml markup extension for Arrays.
     /// </summary>
     [TypeForwardedFrom("PresentationFramework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
     [ContentProperty("Items")]
     [MarkupExtensionReturnType(typeof(Array))]
     public class ArrayExtension : MarkupExtension
     {
+        private readonly ArrayList _arrayList = new ArrayList();
```

Comment:
Reason for converting this to `readonly` ?

Reply:
It is never reassigned